PowerPointLabs/PowerPointLabs/Utils/GraphicsUtil.cs

```diff
@@ -28,9 +28,14 @@ internal static class GraphicsUtil
 
         #region Constants
         private static readonly Object FileLock = new object();
-        public const float PictureExportingRatio = 96.0f / 72.0f;
+        public static float PictureExportingRatio = 330.0f / 72.0f;
         private const float TargetDpi = 96.0f;
         private static float dpiScale = 1.0f;
+
+        // Picture exporting ratios
+        private const float PictureExportingRatioHigh = 330.0f / 72.0f;
+        private const float PictureExportingRatioCompressed = 96.0f / 72.0f;
+
         // Heuristics for image compression obtained through testing
         private const long targetCompression = 75L;
```

Comment:
I realised that these variables are lower-case while the above const float is upper-case for the first letter, could you standardise them according to https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-CSharp.adoc ?

Reply:
Done! Changed to camel case for private variabels.